src/main/charts/confluence/values.yaml

```diff
@@ -77,6 +77,14 @@ confluence:
     # -- The number of consecutive failures of the Confluence container readiness probe before the pod fails readiness checks
     failureThreshold: 30
 
+  accessLog:
+    # -- True if access logging should be enabled.
+    enabled: false
```

Comment:
For reference, the access logs are disabled in the docker image by default.
`ATL_TOMCAT_ACCESS_LOG (default: false)`
https://hub.docker.com/r/atlassian/confluence-server/

Reply:
I've just changed the `values.yaml` file to default this to `true`, since access logs are always enabled for Jira and Bitbucket. Te Docker images may be inconsistent, but we can paper over the cracks here.